saleor/app/tests/test_validators.py

```diff
@@ -191,27 +183,28 @@ def test_new_tab_relative_url_without_app_url(app_manifest):
         _clean_extension_url(extension, manifest_data=app_manifest)
 
 
-def test_clean_extension_url_https_only(settings):
-    # given
+def test_clean_extension_url_https_validation_removed(settings):
+    # given - HTTPS-only validation for POST method was removed
     settings.ENABLE_SSL = True
 
-    # when & then
-    with pytest.raises(ValidationError):
-        _clean_extension_url(
-            {
-                "url": "http://app.example.com/page",
-                "target": AppExtensionTarget.NEW_TAB,
-                "options": {"newTabTarget": {"method": "POST"}},
-            },
-            {
-                "tokenTargetUrl": "https://app.example.com",
-                "appUrl": "https://app.example.com",
-            },
-        )
+    # when - should not raise ValidationError anymore
+    _clean_extension_url(
+        {
+            "url": "http://app.example.com/page",
+            "target": AppExtensionTarget.NEW_TAB,
+            "options": {"newTabTarget": {"method": "POST"}},
+        },
+        {
+            "tokenTargetUrl": "https://app.example.com",
+            "appUrl": "https://app.example.com",
+        },
+    )
 
+    # then - no error is expected since validation was removed
```

Comment:
Valdiation was removed int this PR, but in 2 years, this description will be confusing. 
Can you go through your changes, and  drop the infos like `no ValidationError anymore` or `was remvoed`, 
These have a context in terms of your PR, but not in terms of the whole code.